bindings/javascript/__test__/better-sqlite3.spec.mjs

```diff
@@ -189,6 +189,86 @@ test("Test raw(): Rows should be returned as arrays", async (t) => {
   t.deepEqual(rows[1], ["Bob", 24]);
 });
 
+test("Test expand(): Columns should be namespaced", async (t) => {
+  const expandedResults = [
+    {
+      users: {
+        name: "Alice",
+        type: "premium",
+      },
+      addresses: {
+        userName: "Alice",
+        type: "home",
+        street: "Alice's street",
+      },
+    },
+    {
+      users: {
+        name: "Bob",
+        type: "basic",
+      },
+      addresses: {
+        userName: "Bob",
+        type: "work",
+        street: "Bob's street",
+      },
+    },
+  ];
+
+  let regularResults = [
+    {
+      name: "Alice",
+      street: "Alice's street",
+      type: "home",
+      userName: "Alice",
+    },
+    {
+      name: "Bob",
+      street: "Bob's street",
+      type: "work",
+      userName: "Bob",
+    },
+  ];
+
+  const [db] = await connect(":memory:");
+  db.prepare("CREATE TABLE users (name TEXT, type TEXT)").run();
+  // prettier-ignore
+  db.prepare("CREATE TABLE addresses (userName TEXT, street TEXT, type TEXT)")
+    .run();
+  // prettier-ignore
```

Comment:
@LeMikaelF Let's do one PR just to run prettifier once on these tests and then drop all this noise?